src/main/kotlin/Network.kt

```diff
@@ -0,0 +1,29 @@
+class Network() {
+val capacity = 5
+    fun tallyUpCars(ListOfInterestToDrive:MutableList<Int>) : Int {
+        var ListOfCarsThatWantToDrive:MutableList<Int> = mutableListOf()
+        for (interestToDrive in ListOfInterestToDrive){
+            if (interestToDrive==1){
+                ListOfCarsThatWantToDrive.add(0)
+            }
+
+        }
+        val numberOfCarsThatWantToDrive = ListOfCarsThatWantToDrive.size
+        return numberOfCarsThatWantToDrive
+    }
+
```

Comment:
But this part actually does exactly what you mean, doesn't it? It compares the network capacity with the CarsThatWantToDrive. You would just need to revamp it a bit so that maybe it doesn't just only return an integer. I think in the task it only says that the interestToDrive should be in the .csv-file, adding the value of the CarsThatAreAllowedToDrive into it aswell would be overkill, seeing that it's being processed here.